worksheet_B/clmg/README.txt

```diff
@@ -1,179 +1,179 @@
---------------------------------------------------------------------------------
```

Comment:
You renamed the `CImg` directory (uppercase `i`) to `clmg` (lowercase L) which prevented your program from compiling... I'm not sure how that could have happened?